views/default/input/userpicker.php

```diff
@@ -57,6 +86,6 @@
 </div>
 <script>
 	require(['elgg/UserPicker'], function (UserPicker) {
-		UserPicker.setup('.elgg-user-picker[data-name="<?php echo $name ?>"]');
+		UserPicker.setup('.elgg-user-picker[data-name=<?= json_encode($name) ?>]');
```

Comment:
This is fine. In an ideal world we'd use CSS.escape or its [polyfill](https://github.com/mathiasbynens/CSS.escape/blob/master/css.escape.js).